src/consts.py

```diff
@@ -6,13 +6,12 @@
 import os
 import aqt
 from typing import NewType
-from anki import buildinfo
 from aqt.utils import tr
 from aqt.qt import QT_VERSION_STR
 
 CURRENT_VERSION = '1.3.6'
 
-CURRENT_ANKI_VER = int(buildinfo.version.replace('2.1.', ''))
+CURRENT_ANKI_VER = 66 # hardcoding the anki version to the last 2.1.x
```

Comment:
a workaround for not having a problem with parsing the new anki version. Anki changed versioning scheme.